Source/Extensions/Blazorise.DataGrid/EventArguments.cs

```diff
@@ -162,6 +162,41 @@ public class DataGridReadDataEventArgs<TItem> : EventArgs
         public CancellationToken CancellationToken { get; set; }
     }
 
+    /// <summary>
+    /// Provides all the information for filtered data items.
+    /// </summary>
+    /// <typeparam name="TItem">Type of the data model.</typeparam>
+    public class DataGridFilteredDataEventArgs<TItem> : EventArgs

```

Comment:
Love these! Easy to extend without breaking API, in the future! 👍

Reply:
Yeah, we should try to make it similar in other places, where it makes sense.